src/applications/disability-benefits/all-claims/pages/militaryHistory.js

```diff
@@ -24,7 +24,20 @@ const validateAge = (
   }
 };
 
+const validateSeparationDate = (errors, dateString) => {
+  const ninetyDaysFromToday = moment().add(90, 'days');
```

Comment:
Seems like this should be a comparison of a date with a specific time, 11:59 p.m.? I was just thinking about time zone differences.